## Add additional servos here by adding the pin, neutral and 
## range for each servo. Four servos are setup, as the Exomy
## Hat has 4 open slots
## The servo needs to be registered as well in the "src/servo.py"
## (exept the 4 existing ones)
## Be aware that adding a servo adds a value to the
## servo_angles message in "msg/ServoCommands.msg". You
## might need to adapt further code for example in the WebGui
## to control the servos.
## As every new message, deletes the settings for the other servos
## you need to set this in the coding.

# Pin numbers for servos on PWM board (99 stands for un-used servos)
pin_servo_1: 8
pin_servo_2: 99
pin_servo_3: 99
pin_servo_4: 99

## PWM values
# Those values highly depend on the motors and pwm board used
# Check out the wiki for further explanation

# PWM values for the neutral servo position
servo_pwm_neutral_1: 290
servo_pwm_neutral_2: 300
servo_pwm_neutral_3: 300
servo_pwm_neutral_4: 300

# PWM range around servo neutral position
servo_pwm_range_1: 183
servo_pwm_range_2: 180
servo_pwm_range_3: 180
servo_pwm_range_4: 180